### PR TITLE
Rename column support in Hive Connector

### PR DESCRIPTION
**Pull request by zhenxiao**
@nileema @electrum 
added rename column support in Hive Connector

### CODE REVIEW

**Comment by nileema** [comment]
I'll review this.

File: presto-spi/src/main/java/com/facebook/presto/spi/ConnectorMetadata.java [hunk 1]
```diff
@@ -113,10 +113,7 @@ default ConnectorTableLayout getTableLayout(ConnectorTableLayoutHandle handle)
     /**
      * Rename the specified column
      */
-    default void renameColumn(ConnectorTableHandle tableHandle, ColumnHandle source, String target)
```
Comment (electrum) [inline presto-spi/src/main/java/com/facebook/presto/spi/ConnectorMetadata.java]: This should not change

File: presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java [hunk 2]
```diff
@@ -165,6 +165,12 @@ public void commitCreateTable(ConnectorOutputTableHandle tableHandle, Collection
     }
 
     @Override
+    public void renameColumn(ConnectorTableHandle tableHandle, ColumnHandle source, String target)
+    {
+        throw new PrestoException(NOT_SUPPORTED, "This connector does not support renaming columns");
```
Comment (electrum) [inline presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java]: Only the Hive connector should be changed for this PR

**Comment by zhenxiao** [comment]
@nileema @electrum Thanks, get comments addressed

**Comment by zhenxiao** [comment]
@nileema @electrum I put this into:
https://github.com/facebook/presto/pull/3037
comments and suggestions are appreciated

**Pull request closed without merging**